taskboard/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .interface import TaskBoard
```

Comment:
what are users supposed to import in order to get the whole interface?
ideally it is a single import?

Reply:
fixed so that `import taskboard` gives users everything.